docs/release/trg-0/trg-9-01.md

```diff
@@ -0,0 +1,30 @@
+---
+title: TRG 9.01 - KIT Getting started
+---
+
+| Status     | Created      | Post-History                           |
+|------------|--------------|----------------------------------------|
+| Draft      | 19-Feb-2024  | Initial contribution                   |
+
+## Why
+
+This TRG serves to maintain a **consistent structure** and ensure **content quality** for the stakeholders.
+
+KIT means Keep It Together. A KIT contains all the artifacts that are relevant to participate in the Catena-X data space as an operator, data provider or solution / app provider. These stakeholders have one place with all the information they need. For example an explanation of an API, the specification of it and the deployment instructions. The goal of a KIT should be that you have no need to visit any other information source.
+
+A detailed explanation can be found here:
+[KIT General Introduction](https://eclipse-tractusx.github.io/Kits)
+
+## Description
+
+For providing a KIT to the Eclipse project Tractus-X please be aware of the "getting started" as a contributor.
+
+Prerequisites: [Tractus-X getting started](https://eclipse-tractusx.github.io/docs/oss/getting-started)
+
+- **TRG 8.02 KIT Graduation** A KIT is always structured the same way and follows three graduation stages
+- **TRG 8.03 KIT Artifacts** Each of the artifacts in a KIT are explained with examples
```

Comment:
TRG 8 is related to security and not kits

Reply:
You absolutely right - Will be resolved with the next PR :)